cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/filters/VendorReportFilter.java

```diff
@@ -4,6 +4,10 @@
 
 import java.util.Map;
 
+/**
+ * @deprecated by {@link ResellingReportFilter}
+ */
+@Deprecated
 public class VendorReportFilter extends ReportFilter
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)